src/pages/pattern-library.tsx

```diff
@@ -66,6 +68,15 @@ function Page() {
               </li>
             </ul>
           </section>
+
+          <section className="grid-section grid-content">
+            <h2 className="title-subsection">Inputs</h2>
+            <ul className="list-horizontal">
+              <li style={{ width: '50%' }}>
```

Comment:
I'd remove this width definition so we can see at the pattern library that the default behavior of the `SearchInput` component is to fill the whole width.

Reply:
Actually, in this case the search input width depends on the parent :/ (It will fits its parent width, so I will just change it for 100% for now). Thanks for reviewing :)